matrix/spiral_print.py

```diff
@@ -10,8 +10,8 @@
 
```

Comment:
```suggestion
from collections.abc import Iterable
```
This covers `dict`, `list`, `set`, `tuple`, etc. so we can handle (almost) anything the caller sends our way.